src/test/java/gov/healthit/chpl/aqa/stepDefinitions/Hooks.java

```diff
@@ -38,7 +45,7 @@
     /**
      * Launch ChromeDriver.
      */
-    @Before
+    @Before("@Regression")
```

Comment:
Does this mean that UI tests can only be run as part of the @Regression suite of tests?

Reply:
Yes we separated tags for UI and API test.
UI test will be tagged as @Regression and API will be tagged as @RegressionAPI

Reply:
Right, but what if I don't want to run Regression tests at all? What if I just want to run a specific feature file, or a specific test tagged @WIP? Will these Before and After steps not run if I don't also include the @Regression tag?

Reply:
Yes you need to add those tags to this before and after.

Reply:
If that's the case then the README.md file will need to be updated to indicate that, and the various samples of how to run "a feature file" or "a tag" or some combination of both or whatever will need to be figured out too

Reply:
I ll work on updating that

Reply:
I have pushed work around for this issue. now before and after will ignore tags for RegressionAPI and will run for all other tags. Readme is also updated to reflect that